### ISSUE TITLE
Feature: configurable modifier ordering...

### ISSUE THREAD

**Issue by jshier** [nicklockwood/SwiftFormat#618]
In 0.44.8, modifier ordering was changed "to match SwiftLint". Given that SwiftLint has configurable modifier ordering, SwiftFormat's default behavior now conflicts with my preferred ordering. There doesn't seem to be any way to control to the ordering of modifiers in SwiftFormat, so in order to fully match SwiftLint here, a configurable ordering is needed. Is this possible?

**Comment by nicklockwood** [comment]
@jshier apologies, I probably shouldn't have included that change in a point release. It's certainly possible to add ordering customization.

**Comment by nicklockwood** [comment]
@jshier I've added a `--specifierorder` argument in 0.44.9

**Comment by jshier** [comment]
Thanks!

**Comment by fabianbraach** [comment]
What am I doing wrong? I added

`--specifierorder public,override`

to my `.swiftlint` file. But it still changes my methods to `override public func..`

**Comment by nicklockwood** [comment]
@fabianbraach it seems that I'm sorting the arguments alphabetically, which didn't show up in the tests. 🤦‍♂️

You can fix it by using:

```swift
--specifierorder internal,override
```

Or, to get the pre-0.44.8 behavior, use:

```swift
--specifierorder convenience,override
```

**Comment by fabianbraach** [comment]
@nicklockwood Thanks, this works for now 🙂

**Comment by itaiferber** [comment]
@nicklockwood I just hit that sorting when specifying a long `--specifierorder` myself; thought I'd been going crazy! Uniquing with a `Set` should be fine but the original order should be preserved. I'm assuming you're using this issue to track that?

(For now we'll stick to `0.44.7`)

**Comment by nicklockwood** [comment]
@itaiferber it's already fixed on develop - fix will be included in the next release.

**Comment by itaiferber** [comment]
@nicklockwood Ah, excellent; thanks for the quick response!

**Comment by nicklockwood** [comment]
Fixed in 0.44.10.

**Comment by itaiferber** [comment]
@nicklockwood Confirmed that this resolves the issue for us. Thanks!
